src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -68,10 +64,7 @@
     <Compile Remove="coreclr\Math\Functions\Double\Asinh.cs" />
     <Compile Remove="coreclr\Math\Functions\Double\Atanh.cs" />
     <Compile Remove="coreclr\Math\Functions\Double\Cbrt.cs" />
-    <Compile Remove="coreclr\Math\Functions\Single\Acosh.cs" />
-    <Compile Remove="coreclr\Math\Functions\Single\Asinh.cs" />
-    <Compile Remove="coreclr\Math\Functions\Single\Atanh.cs" />
-    <Compile Remove="coreclr\Math\Functions\Single\Cbrt.cs" />
+    <Compile Remove="coreclr\Math\Functions\Single\**" />
```

Comment:
We have this here but then again in the other ItemGroup at Line 76-78 below under:
```csproj
<ItemGroup Condition=" '$(TargetFrameworkIdentifier)' == '.NETFramework' Or ('$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(_TargetFrameworkVersionWithoutV)' &lt; '3.0')">
```
How come?

Also, we use wildcards for Single but not for Double, why?

Reply:
Also, are we use we want to remove all of them for Single as well? I would have thought most of them are fine to keep (and only the 4 listed previously need to be excluded):
https://github.com/dotnet/performance/tree/master/src/benchmarks/micro/coreclr/Math/Functions/Single

Reply:
The Single API does not exist for Full .NET Framework. ILogB, Log2, ScaleB use APIs added in .NET Core 3.0.

So we exclude all Single tests for Full Framework and those which use .NET Core 3.0 API for non .NET Core 3.0